content/articles/beginners-intro-to-generative-modeling/index.md

```diff
@@ -0,0 +1,102 @@
+
+### Discriminative vs Generative Modeling
+
+Machine learning models can broadly be categorized into two categories, Discriminative and Generative Models. Before we compare both of them, let's talk about the discriminative models. In simple terms, as its name suggests, the discriminative model aims to discriminate between multiple data instances. It takes input data for training and makes predictions for the unseen data. Most of the classification and regression techniques fall under this category.
+To understand it mathematically, let's take an example of a set of data instances $X$ and a corresponding set of labels $Y$.
+
+![discriminative-vs-generative](discriminative-vs-generative.png)
+
+[Image source](https://www.analyticsvidhya.com/blog/2021/07/deep-understanding-of-discriminative-and-generative-models-in-machine-learning/)
+
+The discriminative model aims to capture $p(Y|X)$, that is, the conditional probability of $Y$ given $X$. In contrast, the generative model aims to capture joint probability $p(X, Y)$, or just $p(X)$ if there is no $Y$ present. 
+
+### Why Generative Modeling?
+
+>*The Future of AI is Generative, not Discriminative.*
+> 
+> Steve Omohundro, Research Scientist at Facebook 
+
+Generative modeling is a most underrepresented topic in machine learning. However, still, a lot of research is going on this topic. This section will discuss why generating data is essential and how it can transform the future of machine learning by taking a few interesting use-cases.
+
+The most trending topic in generative modeling is GAN. It is the most effective technique when it comes to generating data. We will talk more about GAN in the next section. For now, let's stick to its applications. GANs have transformed the world of deep learning and are considered the most remarkable invention in artificial intelligence. From generating portraits to the whole new virtual world, GAN never fails to surprise. [Here](https://www.cnet.com/news/ai-made-portrait-sells-at-christies-auction-for-432500/) is one such exciting story of GANs. In 2018, a French art collective named Obvious used GAN to generate a portrait sold for half a million dollars. 
```

Comment:
How do you say it is the most effective technique to generate data. Cite sources?